themes/hive/layouts/partials/explain.html

```diff
@@ -0,0 +1,43 @@
+<!---
+  Licensed to the Apache Software Foundation (ASF) under one
+  or more contributor license agreements.  See the NOTICE file
+  distributed with this work for additional information
+  regarding copyright ownership.  The ASF licenses this file
+  to you under the Apache License, Version 2.0 (the
+  "License"); you may not use this file except in compliance
+  with the License.  You may obtain a copy of the License at
+
+  http://www.apache.org/licenses/LICENSE-2.0
+
+  Unless required by applicable law or agreed to in writing,
+  software distributed under the License is distributed on an
+  "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+  KIND, either express or implied.  See the License for the
+  specific language governing permissions and limitations
+  under the License. -->
+
+<explain>
+    <div id="explain" data-termynal data-termynal data-ty-typeDelay="40" data-ty-lineDelay="70"><span data-ty="input">jdbc:hive2://localhost:10001/> explain select * from test;</span>
+
+        <span data-ty>+---------------------------------------------+</span>
+        <span data-ty>                     Explain                      </span></span>
+        <span data-ty>+---------------------------------------------+</span>
+        <span data-ty>STAGE DEPENDENCIES:                               </span>
+        <span data-ty>  Stage-0 is a root stage                         </span>
+        <span data-ty>STAGE PLANS:                                      </span>
+        <span data-ty>  Stage: Stage-0                                  </span>
+        <span data-ty>    Fetch Operator                                </span>
+        <span data-ty>      limit: -1                                   </span>
+        <span data-ty>      Processor Tree:                             </span>
+        <span data-ty>        TableScan                                 </span>
+        <span data-ty>          alias: test                        </span>
+        <span data-ty>          Statistics: Num rows: 16 Data size: 2880 Basic stats: COMPLETE Column stats: NONE</span>
+        <span data-ty>          Select Operator                         </span>
+        <span data-ty>            expressions: id (type: int), name (type: string), intid (type: int)</span>
+        <span data-ty>            outputColumnNames: _col0, _col1, _col2</span>
+        <span data-ty>            Statistics: Num rows: 16 Data size: 2880 Basic stats: COMPLETE Column stats: NONE</span>
+        <span data-ty>            ListSink                              </span>
+        <span data-ty>+---------------------------------------------+</span>
+    </div>
```

Comment:
Since we are talking about CBO here it would be better to change the example to use `EXPLAIN CBO` and maybe use a less trivial query with 2-3 joins, aggregation, and filter. Also using a well-known datataset (such as TPC-H or TPC-DS) would make this more realistic.

Reply:
I have updated this with a join query on tpcds tables.